### PR TITLE
make shit prettier

### PR DESCRIPTION
**Pull request by kehphin**
screen shot 2015-03-16 at 4 21 36 pm

### CODE REVIEW

**Comment by davlin93** [comment]
No bootstrap for file input?

**Comment by kehphin** [comment]
i cant get it to work lol you need to do some janky css manipulation and its not that important

File: stimulus.html [hunk 1]
```diff
@@ -29,14 +28,9 @@
           </div>
           <div class="collapse navbar-collapse" id="navbar-collapsed">
             <ul class="nav navbar-nav">
-              <li><a href="#">Home</a></li>
-              <li><a href="#">About</a></li>
-              <li><a href="#">Catalog</a></li>
-              <li class="active"><a href="#">Profile <span class="sr-only">Profile</span></a></li>
```
Comment (tonyjhuang) [inline stimulus.html:35]: so we're NOT letting user's create profiles? wtf is this app even

**Comment by tonyjhuang** [comment]
:shipit:  (shipit)

**Pull request merged**